mozilla/bedrock
# [fix bug 1448785] Videos on /firefox/ should only load on click

## Description
- Lazy loads videos on `/firefox` on click to improve page load performance & overall weight.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1448785

## Testing
- Ensure videos load & play with no regressions.
- Check responsiveness.
